make rpm does not work on Fedora rawhide

## Environment Information

- PMDK package version(s): 1.9, current master
- OS(es) version(s): Fedora Rawhide ([docker image](https://github.com/pmem/pmemkv/blob/master/utils/docker/images/Dockerfile.fedora-rawhide))

## Please provide a reproduction of the bug:

`make -j$(nproc) BUILD_PACKAGE_CHECK=n rpm`

While preparing rpm packages (using [PMDK installation script from pmemkv's repo](https://github.com/pmem/pmemkv/blob/7996625bb149bf393bd34e2c902fdf53bc0007bf/utils/docker/images/install-pmdk.sh) and my fix applied: #4937 ) I spotted a lot of messages in the log: "bfd plugin: invalid symbol type found" (first time [here](https://travis-ci.org/github/pmem/pmemkv/jobs/716964800)) and an error ([full log](https://github.com/pmem/pmdk/files/5058581/fedora_rawhide_cut.log)):
```
...
bfd plugin: invalid symbol type found
collect2: error: ld returned 1 exit status
make[3]: *** [../Makefile.inc:242: ../nondebug/libpmempool/../libpmempool.so.1.0.0] Error 1
make[3]: Leaving directory '/pmdk/rpmbuild/BUILD/pmdk-1.7+git1434.g0dcdecfdb/src/libpmempool'
make[2]: *** [Makefile:145: libpmempool] Error 2
make[2]: Leaving directory '/pmdk/rpmbuild/BUILD/pmdk-1.7+git1434.g0dcdecfdb/src'
make[1]: *** [Makefile:56: all] Error 2
make[1]: Leaving directory '/pmdk/rpmbuild/BUILD/pmdk-1.7+git1434.g0dcdecfdb'
error: Bad exit status from /var/tmp/rpm-tmp.RNupgY (%build)

RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.RNupgY (%build)
make: *** [Makefile:123: rpm] Error 1
The command '/bin/sh -c ./install-pmdk.sh rpm' returned a non-zero code: 2
```

## How often bug is revealed:
always

--- a/utils/pmdk.spec.in
+++ b/utils/pmdk.spec.in
@@ -703,6 +703,7 @@ a device.
 
 
 %build
+%define _lto_cflags %{nil}
 # For debug build default flags may be overridden to disable compiler
 # optimizations.
 CFLAGS="%{optflags}" \

